library/crates/Cargo.toml

```diff
@@ -7,109 +25,33 @@ path = "" # ignored by cargo raze
 
 [dependencies]
 alcoholic_jwt = "=4091.0.0"
-async-trait = "0.1.59"
 axum = { version = "=0.5.15", features = ["ws"] }
-axum-server = { version = "=0.4.4", features = ["tls-rustls"]}
-chrono = "=0.4.22"
+chrono = "=0.4.23"
 clap = "=4.0.26"
-clap_derive = "=4.0.21"
 crossbeam = "=0.8.2"
 cucumber = "=0.15.2"
-cucumber-codegen = "=0.15.3"
-cucumber-expressions = "=0.2.1"
 cxx = "=1.0.59"
-derivative = "=2.2.0"
-enum_dispatch = "=0.3.8"
+fluvio-helm = "=0.4.3"
 futures = { version = "=0.3.21", features = ["executor", "thread-pool"] }
 itertools = "=0.10.3"
-log = "=0.4.8"
-mime = "0.3.16"
-mime_guess = "2.0.4"
+k8s-openapi = { version = "=0.16.0", features = ["v1_25"] }
+kube = { version = "=0.77.0", features = ["runtime", "derive"] }
+librocksdb-sys = "=0.8.0+7.4.4"
+log = "=0.4.17"
+mime = "=0.3.16"
+mime_guess = "=2.0.4"
 pest = "=2.4.1"
 pest_derive = "=2.4.1"
 prost = "=0.11.0"
-prost-build = "=0.11.1"
-reqwest = { version = "=0.11.12", features = ["json"] }
+prost-build = "=0.11.0"
 regex = "=1.6.0"
-ring = "=0.16.20"
-rocksdb = "=0.17.0"
-rustix = "=0.35.9"
-serde = "=1.0.145"
-serde_json = "1.0"
-serial_test = "0.9.0"
+reqwest = { version = "=0.11.12", features = ["json"] }
+rocksdb = "=0.19.0"
+serde = "=1.0.152"
+serde_json = "=1.0"
+serial_test = "=0.9.0"
 sha2 = "=0.10.6"
-synthez = "=0.2.0"
-synthez-codegen = "=0.2.0"
-synthez-core = "=0.2.0"
-tokio = { version = "=1.20.1", features = ["rt", "rt-multi-thread"] }
+tokio = { version = "=1.24.1", features = ["rt", "rt-multi-thread"] }
```

Comment:
Some crates' versions, such as chrono, were bumped as required by newly added crates and this bump to tokio.